docs/source/kiva/compiled_path.rst

```diff
@@ -0,0 +1,25 @@
+CompiledPath
+============
+
+A path is a collection of geometry that can be drawn in a graphics context with
+coloring and an affine transformation applied to it. It is the basic unit of
+drawing in a graphics context.
+
+Every graphics context instance has a current path which can be manipulated by
+the :ref:`kiva_path_functions`. However, some drawing operations are easier to
+implement with an independent path instance.
+
+An independent path instance can be created in two ways. The first is via the
+:py:meth:`GraphicsContext.get_empty_path` method. The second method is to use
+the :class:`CompiledPath` class imported from the backend being used. The
+interface of a :class:`CompiledPath` instance is the same as the
+:ref:`kiva_path_functions` (modulo :py:meth:`get_empty_path`).
+
+Once you have a path object, it can be drawn by adding it to the graphics
+context with the :py:meth:`GraphicsContext.add_path` method (which adds the path
+to the current path) and then calling any of the :ref:`kiva_drawing_functions`
+which operate on the current path.
```

Comment:
It might be useful to have some short example code / visual here.

Such as this from Corran's LnL:
<img width="1326" alt="Screen Shot 2021-03-02 at 9 18 21 AM" src="https://user-images.githubusercontent.com/36972686/109670844-250ea080-7b39-11eb-849a-352f56e51395.png">

or just the existing compiled_path.py demo in kiva that draws stars.

Reply:
Yup. That snippet of code makes for a good example.